client/src/components/Doctor/DoctorDashboard.tsx

```diff
@@ -3,34 +3,75 @@ import React, { useContext, useState, useEffect } from 'react';
 import { useNavigate } from 'react-router-dom';
 import { AuthContext } from '../../providers/AuthProvider';
 import { HealthContext } from '../../providers/HealthProvider';
+const AWS = require('aws-sdk');
+import * as CryptoJS from 'crypto-js';
 
 const DoctorDashboard: React.FunctionComponent<{}> = () => {
   const { user, role, logout } = useContext(AuthContext);
-  const { handleRaiseRequest } = useContext(HealthContext);
-  const [hcProvider, setHCProvider] = useState<string>('');
+  const {fetchHealthCareProviders, providers, handleRaiseRequest} = useContext(HealthContext);
+  const [hcProvider, setHCProvider] = useState<any>('');
+  const [providerId, setProviderId] = useState<any>('');
   const { fetchAllDoctors, doctorList } = useContext(HealthContext);
-
+  const s3 = new AWS.S3({
+    accessKeyId: process.env.REACT_APP_AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.REACT_APP_AWS_SECRET_ACCESS_KEY,
+    region: process.env.REACT_APP_AWS_REGION
+  });
```

Comment:
these are senstive information, if leaked will have monetary imapct. should not be available on front end. please refactor to store them via node js api